rpc_jobs/lem_multi_node_aio.yml

```diff
@@ -99,16 +102,22 @@
               tempest = load 'pipeline_steps/tempest.groovy'
               holland = load 'pipeline_steps/holland.groovy'
               maas = load 'pipeline_steps/maas.groovy'
+              kibana = load 'pipeline_steps/kibana.groovy'
           }}
           dir("rpc-gating/playbooks") {{
             writeFile file: "inventory/hosts", text: "[job_nodes]\n${{INSTANCE_NAME}} ansible_host=${{INSTANCE_IP}}\n"
             common.install_ansible()
           }}
           ssh_slave.connect()
           node(INSTANCE_NAME) {{
-            sh """
-               lvcreate -n aio -L250G lxc
-            """
+            common.conditionalStage(
```

Comment:
I did, but I think the job template will need a fair bit of refactoring once https://github.com/rcbops/rpc-gating/pull/149 lands so I figured we could extract some of the function calls to separate groovy scripts once we perform those updates.